### PR TITLE
#2559. Add tests for adding body to static members

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

**Review by eernstg — APPROVED** [review]
LGTM

File: LanguageFeatures/Augmentation-libraries/augmenting_declarations_A05_t04_lib.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+// SharedOptions=--enable-experiment=macros
+
+part of  'augmenting_declarations_A05_t04.dart';
+
+augment class C {}
+
+augment mixin M {}
+
+augment enum E {}
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/augmenting_declarations_A05_t04_lib.dart:23]: Perhaps?:
```suggestion
augment enum E { e1 }
```

**Review by eernstg — COMMENTED** [review]
Looks good, with one likely typo!

File: LanguageFeatures/Augmentation-libraries/augmenting_declarations_A05_t04_lib.dart (hunk 1 above)
Comment (sgrekhov) [reply LanguageFeatures/Augmentation-libraries/augmenting_declarations_A05_t04_lib.dart]: Of course. Thank you! Fixed. PTAL.

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**